torchsynth/module.py

```diff
@@ -98,7 +117,7 @@ def to_buffer_size(self, signal: Signal) -> Signal:
     def seconds_to_samples(self, seconds: T) -> T:
         # Do we want this?
         # assert seconds.ndim == 1
-        return torch.round(seconds * self.sample_rate).int()
+        return torch.round(seconds * self.sample_rate)
```

Comment:
Better to keep things as floats

Reply:
![image](https://user-images.githubusercontent.com/65918/112719505-93a50b00-8ef9-11eb-8944-799c1379581a.png)

if this is the only time we use it, to create ramps, maybe we don't even want to round it.

Reply:
And just make a comment explaining that

Reply:
Yup, I like that idea